### DISCUSSION TITLE
Support/Documentation for Vanilla users of CDS

### DISCUSSION THREAD

**Discussion by alexandrutatarciuc** [carbon-design-system/carbon#11430] [Help]
My team is creating a design system built on top of CDS. We are building native Blazor components in accordance with CDS guidelines (+ our own styling like colors, fonts, things that would represent our company). Our main guidance until v11 was this website https://the-carbon-components.netlify.app/. It looks like it isn't maintained anymore after the v11 release. On carbondesignsystem.com there used to be a button that forwarded to the vanilla HTML/CSS component implementation website
![image](https://user-images.githubusercontent.com/56069835/168993976-f2bb9bb1-edcd-4bce-868f-a7e771e5b62a.png)
Now the only way to see how you structure your HTML for components, what CSS classes you utilize for styling, and what aria-label attributes you specify is by merely inspecting the page on your react storybook https://react.carbondesignsystem.com/

Will the website be updated in the future?
Is there a new website for this that isn't documented anywhere?

Thank you in advance.

P.S. I'd like to apologize for creating this discussion here, inasmuch as it is more related to carbon-website which doesn't have a discussions tab.

**Comment by joshblack** [comment]
Hi there @alexandrutatarciuc! 👋 

This is a great point, in v11 the Vanilla version of components are officially out of scope of our team and unfortunately will need to be community maintained. The reference for components will be the storybook in React.

Let me know if there is a format we could provide that would be helpful for you all so you don't have to dig in and figure this out manually. I'd be happy to help out!
